Bitkit/Views/Transfer/SpendingAmount.swift

```diff
@@ -10,20 +10,16 @@ struct SpendingAmount: View {
 
     @StateObject private var amountViewModel = AmountInputViewModel()
     @State private var isLoading = false
-    @State private var maxSendableAmount: UInt64?
-    @State private var maxTransferAmount: UInt64 = 0
+    @State private var maxTransferAmount: UInt64?
 
-    var amountSats: UInt64 {
+    private var amountSats: UInt64 {
         amountViewModel.amountSats
     }
 
-    // Calculate the maximum amount that can be transferred
-    private var availableAmount: UInt64 {
-        return maxSendableAmount ?? UInt64(wallet.spendableOnchainBalanceSats)
-    }
-
-    private var transferValues: TransferValues {
-        transfer.calculateTransferValues(clientBalanceSat: amountSats, blocktankInfo: blocktank.info)
+    private var isValidAmount: Bool {
+        guard amountSats > 0 else { return false }
+        guard let max = maxTransferAmount, max > 0 else { return false }
```

Comment:
Available amount should be the total onchain balance minus the network fee, like before

Reply:
But the the available and max amount mismatch

Reply:
So the issue is before, the Max could be more than the available, which really didn't make sense I think (the Figma shows the opposite case which is ok)